tests/phpunit/tests/http/http.php

```diff
@@ -9,6 +9,9 @@ class Tests_HTTP_HTTP extends WP_UnitTestCase {
 	const FULL_TEST_URL = 'http://username:password@host.name:9090/path?arg1=value1&arg2=value2#anchor';
 
 	/**
+	 * @since 3.7.0 Test method introduced.
```

Comment:
```suggestion
```
Unless I'm missing something, we don't generally add these tags for individual test methods. Is this helpful here? 🙂

Reply:
Myself and Peter discussed this and as the `@ticket` was needed for this patch, we felt it might give the false impression that this ticket introduced the test method.

The test method was introduced in 3.7 when tests we moved to svn.wordpress.org, so we weren't sure of the original ticket and opted for an `@since 3.7.0`.

What do you think @SergeyBiryukov? Is there something that would work better in this scenario?

Reply:
I see, thanks! `@since 3.7.0` does not seem correct though, as the test method is older than that, see the [previous history in the older tests repo](https://core.trac.wordpress.org/log/tests/trunk/tests/http/http.php?action=stop_on_copy&mode=follow_copy&rev=1337&stop_rev=&limit=100).

[[713/tests]](https://core.trac.wordpress.org/changeset/713/tests) appears to be the initial commit, so [#20434](https://core.trac.wordpress.org/ticket/20434) is the original ticket here. I have updated the suggestion 🙂

Reply:
Thanks for tracking that down Sergey, it's very helpful.